### DISCUSSION TITLE
Download Destination

### DISCUSSION THREAD

**Discussion by JohnHBrady** [minbrowser/min#1863] [Ideas]
I'd like to be able to select the destination for downloads. I download a lot and normally pick desktop as my destination. I don't see a way of doing this in Min. Did I miss it?

**Comment by PalmerAL** [comment]
When the download starts, you should see a save dialog that lets you pick a destination - are you not seeing that? Or do you mean that you want to set a default location instead of using the dialog each time?

**Reply by JohnHBrady** [reply]
I've been using the dialog. Being able to set a default destination would be nice. Not essential obviously, but an option in other browsers.
This is a minimalist browser, so I know it's worthwhile resisting feature creep. But I'm going ahead with my short list of feature requests.
